doc/source/categorical.rst

```diff
@@ -44,11 +44,26 @@ The categorical data type is useful in the following cases:
 * As a signal to other Python libraries that this column should be treated as a categorical
   variable (e.g. to use suitable statistical methods or plot types).
 
+.. note::
+
+    In contrast to R's `factor` function, categorical data is not converting input values to
+    strings and categories will end up the same data type as the original values.
+
+.. note::
+
+    In contrast to R's `factor` function, there is currently no way to assign/change labels at
+    creation time. Use `categories` to change the categories after creation time.
```

Comment:
I would personally move those back a bit more below. They are specifically about the creation of Categoricals, so I would put it at the end of the 'Object creation' section